src/routing/proxy.rs

```diff
@@ -83,7 +83,7 @@ pub struct ApiProxyState {
     request_body_timeout: Duration,
 }
 
-// Proxies /api/v2/... and /api/v3/... endpoints to the IC
+/// Proxies /api/v2/... and /api/v3/... endpoints to the IC
```

Comment:
Why 3 /// and not 2?

Reply:
`///` is a document comment in Rust from which the docs are generated and hints in IDE e.g.

<img width="378" alt="image" src="https://github.com/user-attachments/assets/44efba91-341e-4d06-93df-55dacd47ba48" />